helpers.go

```diff
@@ -319,3 +322,44 @@ func joinAsArray(cmds []string, emptyArray bool) string {
 	}
 	return str + "]"
 }
+
+// hasMinZypperVersion determines the images's zypper version and compares
+// it with the required version. If the version cannot be determined, it
+// returns false.
+func hasMinZypperVersion(image, version string) bool {
+	requiredVersion := strings.Split(version, ".")
+	requiredMajor, _ := strconv.Atoi(requiredVersion[0])
+	requiredMinor, _ := strconv.Atoi(requiredVersion[1])
+	requiredPatch, _ := strconv.Atoi(requiredVersion[2])
+
+	buf := bytes.NewBuffer([]byte{})
+	id, err := runCommandInContainer(image, []string{"zypper --version"}, true, buf)
+	if err != nil {
+		log.Println("Unable determine zypper version:", err)
+		return false
+	}
+	removeContainer(id)
+
+	re := regexp.MustCompile(`(\d+)\.(\d+)\.(\d+)`)
```

Comment:
I would not go down this path, I already tried that in the past and I got burned. The versions can look way more complicated than that.

The best approach is to use the [same function as rpm](http://rpm.org/api/4.4.2.2/rpmlib_8h.html#86c8c7c0f690d9dfd43ddb9149f7a449).

This is small C program demoing the usage:
```c
#include <rpm/rpmlib.h>

int main() {
  char *v1 = "0.94.5+git.1446971359.1d0909a-12.1";
  char *v2 = "2.0.0";

  printf("%d\n", rpmvercmp(v1,v2));
  return 0;
}
```

To build and run it:
```
$ sudo zypper -n in rpm-devel
$ gcc `pkg-config --libs rpm` main.c -o vercmp
```

This will print `-1` because `v1` is less then `v2`. BTW, v1 is the actual version of a package I've installed on my computer :scream: 

We can just use the same function using `cgo` and introducing a new runtime dep against rpm (totally fine from my POV).